assets/js/ajax-filters.js

```diff
@@ -27,21 +27,6 @@ jQuery( document ).ready( function ( $ ) {
 			xhr[index].abort();
 		}
 
-		if ( ! append ) {
-			$( results ).addClass( 'loading' );
```

Comment:
It's not necessary to add the 'load' class because the onClick handler adds and handles this class on line 207.